public/app/features/apiserver/types.ts

```diff
@@ -42,6 +42,7 @@ export const AnnoKeyFolderId = 'grafana.app/folderId';
 export const AnnoKeyFolderUrl = 'grafana.app/folderUrl';
 export const AnnoKeyMessage = 'grafana.app/message';
 export const AnnoKeySlug = 'grafana.app/slug';
+export const AnnoKeyConversion = 'CONVERSION-0';
```

Comment:
^^^ does not exist and should not exit

(was early development stuff)

Reply:
Done! It was during testing, we added it, but now it uses status field 💯